roles/monit/vars/main.yml

```diff
@@ -0,0 +1,15 @@
+alert_email: <TODO>
```

Comment:
Be sure to write this in the how to setup that this would need to be update/changed if the user wants emails. This would be apart of the smtp steps